applications/ecallmgr/src/ecallmgr_util.erl

```diff
@@ -571,9 +571,34 @@ build_simple_channels(Endpoints) ->
 
 -spec build_bridge_channels(wh_json:objects()) -> bridge_channels().
 build_bridge_channels(Endpoints) ->
-    EPs = endpoint_jobjs_to_records(Endpoints),
+    CWEP = maybe_apply_call_waiting(Endpoints),
+    EPs = endpoint_jobjs_to_records(CWEP),
     build_bridge_channels(EPs, []).
 
+-spec maybe_apply_call_waiting(wh_json:objects()) -> wh_json:objects().
+maybe_apply_call_waiting(Endpoints) ->
+    lists:map(fun call_waiting_map/1, Endpoints).
+
+-spec call_waiting_map(wh_json:object()) -> wh_json:object().
+call_waiting_map(Endpoint) ->
+    CCVs = wh_json:get_value(<<"Custom-Channel-Vars">>, Endpoint, wh_json:new()),
+    case wh_json:is_true(<<"Call-Waiting-Disabled">>, CCVs) of
```

Comment:
Who sets this CCV? I don't see it being set or overridden elsewhere in this pull request.